src/Microsoft.FeatureManagement.AspNetCore/Microsoft.FeatureManagement.AspNetCore.csproj

```diff
@@ -14,6 +14,8 @@
     <Company>Microsoft</Company>
     <PackageLicenseUrl>https://aka.ms/AzureAppConfigurationDotnetCorePublicPreviewEula</PackageLicenseUrl>
     <PackageProjectUrl>https://github.com/Azure/AppConfiguration</PackageProjectUrl>
+    <PackageReleaseNotes>For the latest release notes visit https://aka.ms/MicrosoftFeatureManagementReleaseNotes</PackageReleaseNotes>
```

Comment:
>For the latest release notes visit https://aka.ms/MicrosoftFeatureManagementReleaseNotes [](start = 25, length = 88)

Should there be a comma between `release notes` and `visit`?

Reply:
Not sure, I'll just change it to "The latest release notes can be found at x"

Reply:
Updated.